dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -162,48 +185,35 @@ private Response postWithEndpointState(String endpointId, String url, MediaType
      */
 
     @POST
-    @Path("/generateintegritydata/{params:.*}")
-    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Path("/_generate")
```

Comment:
Let's be explicit here: `_generateintegritydata`

Reply:
https://github.com/dotCMS/core/pull/19273/commits/303cd0bf044e9265865287ea596719a4346e2895#diff-1c1ba904ed03287c1c0b48a6f3971449R182